pkg/glusterfs/driver_test.go

```diff
@@ -169,6 +84,93 @@ func TestDriverSuite(t *testing.T) {
 	sanity.Test(t, cfg)
 }
 
+func handleGETRequest(w http.ResponseWriter, r *http.Request, t *testing.T) {
+	id := uuid.Parse("02dfdd19-e01e-46ec-a887-97b309a7dd2f")
+	if strings.Contains(r.URL.String(), "/v1/peers") {
+		resp := make(api.PeerListResp, 1)
+		resp[0] = api.PeerGetResp{
+			Name: "node1.com",
```

Comment:
nit: a suggestion may be 'node1.gluster.org` instead of 'node1.com' . Please feel free to discard if you dont like it :)

Reply:
done